Tasks/NuGet/nuget.ts

```diff
@@ -0,0 +1,94 @@
+import * as ngToolRunner from "nuget-task-common/NuGetToolRunner";
+import * as nutil from "nuget-task-common/Utility";
+import * as tl from "vsts-task-lib/task";
+global['_vsts_task_lib_loaded'] = true;
```

Comment:
Added a comment, will update to 2.0.4 once the package is published (and when we migrate all of our tasks to that version).